### DISCUSSION TITLE
{shiny.fluent} alternative for `downloadButton()`

### DISCUSSION THREAD

**Discussion by kyle-power** [Appsilon/shiny.fluent#30] [Q&A] [answered]
I am trying to use a `PrimaryButton()` to trigger a download, but this is proving to be a significant challenge. Any pointers for downloads?

Thanks!!

@kamilzyla

**Comment by kamilzyla** [comment, marked as answer]
Hi @kyle-power! We do not support using `PrimaryButton()` as a download button. We can probably find a workaround for this, but it seems to be quite a challenge indeed - I will try to find some time next week to investigate it.

**Reply by kyle-power** [reply]
@kamilzyla Great. I think a download button / function would be a critical component of `{shiny.fluent}`, and I am personally eager to see this feature added in the near future. Your dedication to the project is incredible.  I'll keep an eye on this one :)

**Comment by marekrogala** [comment]
@kamilzyla I just noticed that we don't have a workaround linked here. If we have it somewhere, can you link it here for everyone searching for the solution?

**Reply by jakubsob** [reply]
@marekrogala The workaround can be found here #39
